Starcraft2Client.py

```diff
@@ -793,7 +793,12 @@ def check_game_install_path() -> bool:
         with open(einfo) as f:
             content = f.read()
         if content:
-            base = re.search(r" = (.*)Versions", content).group(1)
+            try:
+                base = re.search(r" = (.*)Versions", content).group(1)
+            except AttributeError:
+                sc2_logger.warning(f"We found {einfo}, but it was empty. Run SC2 through the Blizzard launcher, then "
+                                   f"try again.")
```

Comment:
I would scrap the "We". ~~More shorter is more better.~~ To me, "we" has a personal vibe, but it's the program complaining, not "us" developers.

```suggestion
                sc2_logger.warning(f"Found {einfo}, but it was empty. Run SC2 through the Blizzard launcher, then "
                                   f"try again.")
```